### PR TITLE
arp: add German translation

### PR DESCRIPTION
**Pull request by Slotharmy**
- [ ] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ ] The page(s) have at most 8 examples.
- [ ] The page description(s) have links to documentation or a homepage.
- [ ] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.de/common/arp.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# arp
+
+> Den ARP Cache des Systems anzeigen und manipulieren
```
Comment (pixelcmtd) [inline pages.de/common/arp.md:3]:
```suggestion
> Den ARP Cache des Systems anzeigen und manipulieren.
```

File: pages.de/common/arp.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+> Weitere Informationen: <https://manned.org/arp>.
+
+- Die aktuelle ARP Tabelle anzeigen lassen:
```
Comment (pixelcmtd) [inline pages.de/common/arp.md:6]:
```suggestion
- Zeige die aktuelle ARP Tabelle an:
```

File: pages.de/common/arp.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+`arp -a`
+
+- Den gesamten Cache leeren:
```
Comment (pixelcmtd) [inline pages.de/common/arp.md:10]:
```suggestion
- Leere den gesamten Cache:
```

File: pages.de/common/arp.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+`sudo arp -a -d`
+
+- Einen spezifizierten Eintrag in der Tabelle löschen:
```
Comment (pixelcmtd) [inline pages.de/common/arp.md:14]:
```suggestion
- Lösche einen spezifischen Eintrag in der Tabelle:
```

File: pages.de/common/arp.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+
+`arp -d {{addresse}}`
+
+- Einen Eintrag in der ARP Tabelle erstellen:
```
Comment (pixelcmtd) [inline pages.de/common/arp.md:18]:
```suggestion
- Erstelle einen Eintrag in der ARP Tabelle:
```

**Review by pixelcmtd — CHANGES_REQUESTED** [review]
Are you sure you've read the style guide?

**Review by pixelcmtd — APPROVED** [review]

**Review by acuteenvy — APPROVED** [review]

**Pull request merged**